pubsub/mqtt3/metadata.yaml

```diff
@@ -0,0 +1,67 @@
+# yaml-language-server: $schema=../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: mqtt3
+version: v1
+status: stable
+title: "MQTT3"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-mqtt3/
+authenticationProfiles:
+  - title: "Connection string"
+    description: "The MQTT3 Connection/Authentication."
+    metadata:
+      - name: url
+        type: string
+        required: true
+        sensitive: true
+        description: |
+          Address of the MQTT broker.
+          Use the tcp:// URI scheme for non-TLS communication. 
+          Use the ssl:// URI scheme for TLS communication.
+        example: '"tcp://[username][:password]@host.domain[:port]"'
+metadata:
+  - name: consumerID
+    type: string
+    description: |
+      The client ID used to connect to the MQTT broker. Defaults to the Dapr app ID.
+    example: '"myMqttClientApp"'
+  - name: retain
+    type: bool
+    description: |
+      Defines whether the message is saved by the broker as the last known good value for a specified topic.
+    default: 'false'
+    example: '"true", "false"'
+  - name: cleanSession
+    type: bool
+    description: |
+      Sets the clean_session flag in the connection message to the MQTT broker if "true" (more info).
+    default: 'false'
+    example: '"true", "false"'
+  - name: qos
+    type: number
+    description: |
+      Indicates the Quality of Service Level (QoS) of the message (more info).
+    default: '1'
+    example: '0, 1, 2'
+  - name: caCert
+    type: string
+    description: |
+      Certificate authority certificate, required for using TLS. 
+      Can be secretKeyRef to use a secret reference".
```

Comment:
We explained before many times that `Can be secretKeyRef to use a secret reference".` isn't needed (also why is there a `"` in there?)